libcxx/lib/abi/arm64-apple-darwin.libcxxabi.v1.stable.exceptions.nonew.abilist

```diff
@@ -2508,6 +2509,7 @@
 {'is_defined': True, 'name': '___cxa_guard_abort', 'type': 'I'}
 {'is_defined': True, 'name': '___cxa_guard_acquire', 'type': 'I'}
 {'is_defined': True, 'name': '___cxa_guard_release', 'type': 'I'}
+{'is_defined': True, 'name': '___cxa_init_primary_exception', 'type': 'I'}
```

Comment:
We should be exporting `_ZNSt13exception_ptr31__from_native_exception_pointerEPv` & friends on Darwin too. You're guarding the definition of these functions in the dylib by `_LIBCPP_AVAILABILITY_HAS_INIT_PRIMARY_EXCEPTION` but you shouldn't -- that would solve this issue.

Reply:
Done.